client/my-sites/sidebar/sidebar.jsx

```diff
@@ -326,7 +326,15 @@ module.exports = React.createClass( {
 			linkClass += ' is-paid-plan';
 		}
 
-		let planName = site.plan.product_name_short;
+		let planName = site.plan.product_name_short,
+			labelClass;
+
+		if ( abtest( 'plansUpgradeButton' ) === 'button' && productsValues.isFreePlan( site.plan ) ) {
+			labelClass = 'add-new';
+			planName = 'Upgrade'; // TODO: translate this string if the test is removed
+		} else {
+			labelClass = 'plan-name';
+		}
```

Comment:
> planName should be set to 'Trial' regardless of the abtest / isFreePlan check in the previous condition, and using else if makes it seem (to me) like these conditions are related. 

I guess our brains don't work the same way :).

As far as I know we don't have trials for free plans (an never will) so these conditions are mutually exclusive, which is why I feel using a `if else` helps clarifying that further. Right now it leaves readers in this gray area where they have to use their brain to answer that question.

However, this is not a big deal and it's great to be able to confront opinions. Thanks for the video!